README.md

```diff
@@ -99,6 +99,15 @@ yarn test
 yarn buildProd
 ```
 
+## Preparing a Release
+
+```sh
+VERSION=<VERSION>
+rm -r dist/
```

Comment:
maybe add `-f` in case we later copy this script to CI where we'll maybe start without a dist dir

Reply:
Agreed. Done.